app/javascript/components/ansible-credentials-form/ansible-credentials-form.schema.js

```diff
@@ -0,0 +1,55 @@
+import { componentTypes, validatorTypes } from '@@ddf';
+
+const normalize = (fields, edit) => fields.map(({ component, label, helperText, fields, ...field }) => ({
+  ...field,
+  component,
+  label: __(label),
+  helperText: __(helperText),
+  ...(component === 'password-field' && { edit }),
+  ...(fields && { fields: normalize(fields, edit) }),
+}));
+
+const createSchema = (fields, promise, edit, loadSchema) => ({
+  fields: [
+    {
+      component: componentTypes.TEXT_FIELD,
+      name: 'manager_resource',
+      hideField: true,
```

Comment:
Using `hideField` actually creates a visible text field that can show up if the styling is not properly loaded. It should not be an issue, but it's semantically [better](https://www.w3schools.com/tags/att_input_type_hidden.asp) to use `type: 'hidden'` instead.